retail/interactive-tutorial/RetailSearch.Samples/SearchWithFacetSpec.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_search_product_with_facet_spec]
```

Comment:
Note that everything in between region tags will be reviewed as documentation sample.

Reply:
I think it is better to remove the region tags for now, as we have decided that our main focus is on the interactive tutorials, which will not become a part of the documentation. Later the tutorials code samples can be reused and refactored to fit the doc-samples ruler.
What do you think?

Reply:
Moved region tags according to suggestion below

Reply:
Applied region tags only for sample class inside tutorial by suggestion below